frontend/packages/data-portal/app/components/AuthorLink/AuthorLink.tsx

```diff
@@ -0,0 +1,79 @@
+import { LinkProps } from '@remix-run/react'
+import { ComponentType } from 'react'
+
+import { EnvelopeIcon } from 'app/components/icons'
+import { Link } from 'app/components/Link'
+import { cns } from 'app/utils/cns'
+
+import { ORC_ID_URL } from './constants'
+import orcIdImage from './orcid.png'
+import { AuthorInfo } from './types'
+
+const BASE_ICON_SIZE = 10
+const LARGE_ICON_SIZE = 14
+
+export function AuthorLink({
+  author,
+  large,
+  LinkComponent = Link,
+}: {
+  author: AuthorInfo
+  large?: boolean
+  LinkComponent?: ComponentType<LinkProps>
+}) {
+  const iconSize = large ? LARGE_ICON_SIZE : BASE_ICON_SIZE
+  const content = (
+    <span className="inline">
+      <span
+        className={cns(
+          'inline border-b',
+
+          author.orcid
+            ? [
+                'border-dashed',
+
+                author.primary_author_status
+                  ? 'border-black'
+                  : 'border-sds-gray-500',
+              ]
+            : 'border-transparent',
+        )}
+      >
+        {author.orcid && (
+          <img
+            className="inline"
```

Comment:
super nit - the iconseems to be closer to the dotted line compared to the text. Maybe either add some margin-bottom to the icon style or something?

<img width="532" alt="Screenshot 2024-05-13 at 1 44 37 PM" src="https://github.com/chanzuckerberg/cryoet-data-portal/assets/6309723/49e5b81f-a473-4553-bd2c-d44500b3612b">